Containerfile

```diff
@@ -74,5 +74,10 @@ ARG version=latest
 ENV VERSION=${version}
 ENV QDROUTERD_HOME=/home/skrouterd
 
+ENV ASAN_OPTIONS="disable_coredump=0 detect_odr_violation=0 strict_string_checks=1 detect_stack_use_after_return=1 check_initialization_order=1 strict_init_order=1 detect_invalid_pointer_pairs=2"
+ENV LSAN_OPTIONS="disable_coredump=0 suppressions=/lsan.supp"
```

Comment:
lsan needs to use `ptrace`, which requires capabilities, for which I tend to run container with `--privileged`. Afaik this is no different from gdb, and therefore is not a problem.

one solution (to have asan without lsan) would be to define here `detect_leaks=0`.